astrology.lic

```diff
@@ -72,9 +72,10 @@ class Astrology
     result.compact.select { |data| data['circle'] <= DRStats.circle }
   end
 
-  def check_moons
-    %w[xibar yavash katamba].each do |moon|
-      bput("pow #{moon}", 'roundtime')
+  def check_attunement
```

Comment:
Does this _only_ train Attunement? If so, why is it in the Astrology trainer (and not in the Attunement training script)?

Reply:
perc planets and perc moons train astrology

Reply:
perc mana is attunement training only

Reply:
actually I just added them to finish up the attunement. The actual attunement.lic for moon mages only adds perc mana but with all 4 it drives attunement faster. I also did not know if attunement is called during hunting in which case you would not want the additional calls. perc moons is around 40 seconds at circle 1, perc planets is 20 seconds. perc mana is around 40 seconds. If attunement.lic is actually being called during combat, it may be more advantageous to change it to only call perc with a 10 second or so rt instead of perc mana and allow this to be used for actual moon mage attunement. Also, its a really good way to eat the reset timer on astrology.

Reply:
Im not against it all, I think the change is quite nice

Reply:
is it the actual attunement script called from combat? if so I may modify that to drop moon mage attunement as a 30+ second rt during combat sucks

Reply:
No. `combat-trainer` has two types of Attunement training, neither of which call the `attunement` script:
* Perc: https://github.com/rpherbig/dr-scripts/blob/master/combat-trainer.lic#L1706
* PercMana: https://github.com/rpherbig/dr-scripts/blob/master/combat-trainer.lic#L1910-L1915